collateral_pools/collateral_pool_1/markets/market_1eth.toml

```diff
@@ -0,0 +1,173 @@
+fromCall.func = "owner"
+func = "createMarket"
+args = [
+  "Market ETH",
+  "<%= settings.pool1RusdId %>",
+]
+var.market_1eth.event = "MarketDataUpdated"
+var.market_1eth.arg = 0
+depends = [
+  "invoke.core_register_instrument_passive_perp",
+  "var.passive_pool_1rusd"
+]
+
+[var.market_1eth_vars]
+market1EthId = "<%= (String(settings.market_1eth.id) === '1') ? settings.market_1eth.id : parseUnits('0', 'assert market_1eth_vars failed') %>"
+depends = [
+  "invoke.create_market_1eth"
+]
+
+##################################################
+# Market's Collateral Pool Feature Flags
+##################################################
+[invoke.market_1eth_unpause_cp]
+target = ["CoreProxy"]
+fromCall.func = "getCollateralPoolOwner"
+fromCall.args = ["<%= settings.market1EthId %>"]
+func = "setFeatureFlagAllowAll"
+args = [
+    "<%= keccak256(defaultAbiCoder.encode(['bytes32','uint128'],[keccak256(hexlify('collateralPoolEnabled')),settings.market1EthId])) %>",
+    true
+]
+depends = [
+  "invoke.upgrade_core_proxy", 
+  "var.market_1eth_vars",
+]
+
+[invoke.market_1eth_set_pausers_cp]
+target = ["CoreProxy"]
+fromCall.func = "getCollateralPoolOwner"
+fromCall.args = ["<%= settings.market1EthId %>"]
+func = "setDeniers"
+args = [
+  "<%= keccak256(defaultAbiCoder.encode(['bytes32','uint128'],[keccak256(hexlify('collateralPoolEnabled')),settings.market1EthId])) %>",
+  [
+    "<%= settings.pauser1 %>",
```

Comment:
@avniculae would be nice to have checks in the fork simulation that ensure only the pausers can pause and no-one else to validate this config was applied properly